platform-manageability-agent/internal/comms/comms.go

```diff
@@ -203,13 +199,13 @@ func (cli *Client) RetrieveActivationDetails(ctx context.Context, hostID string,
 		if isProvisioned(string(output)) {
 			req = &pb.ActivationResultRequest{
 				HostId:           hostID,
-				ActivationStatus: pb.ActivationStatus_PROVISIONED,
+				ActivationStatus: pb.ActivationStatus_ACTIVATED,
```

Comment:
Till we ME connects to MPS we should update the status as in ActivationStatus_ACTIVATING and for that we need parse the rpc amtinfo command output.